Makefile

```
@@ -71,9 +71,27 @@ help:
 
 .PHONY: test
-test:
+test: lint-pipes
 	@$(SWIFT) test -c $(BUILD_CONFIGURATION)
 
+# Prevent Foundation's Pipe() from being used when passing fds to Apple Container
+# APIs (createProcess/bootstrap). Apple closes those fds immediately after duping
+# them into the container, causing Pipe.deinit to double-close a reused fd and
+# corrupt NIO's fd table (writev/kevent EBADF crash). Use ProcessPipe instead.
+.PHONY: lint-pipes
+lint-pipes:
```

checkmake (0.3.2)

[warning] Target body for "lint-pipes" exceeds allowed length of 5 lines (11).

(maxbodylength)